### ISSUE TITLE
support setting / jumping to notebook-wide marks in Jupyter mode

### ISSUE THREAD

**Issue by edgimar** [lambdalisue/jupyter-vim-binding#75]
It would be great if it were possible to set marks for a particular notebook that would allow jumping to any particular location within the notebook.  For example, given a notebook with 100 cells in it, if the 20th cell is currently selected, then when in Jupyter mode, typing `ma` should store a mark to that location in the notebook.  Later, to return to that location, one would type ``a`.

It's not clear what the best way to define 'location' is here, but one way would simply be to associate the location with a particular cell (even if this would be confusing if the cell gets moved to a different part of the notebook).

**Comment by lambdalisue** [comment]
I agree that it is great but I think it is a bit hard to implement. Currently I don't really have time to discover the way to do, sending me a PR is really welcome ;-)

Ref: https://github.com/lambdalisue/jupyter-vim-binding/issues/69#issuecomment-215682231
